kitchen-tests/.kitchen.travis.yml

```diff
@@ -35,6 +35,7 @@ platforms:
     pid_one_command: /sbin/init
     intermediate_instructions:
       - RUN yum -y install sudo
+      - RUN yum -y install gcc-c++
```

Comment:
Can you combine the 2 yum lines and add a comment on why we're adding gcc-c++. This reduces the number of layers that have to be created